cosmos/launchpad/construction_metadata.go

```diff
@@ -0,0 +1,35 @@
+package launchpad
+
+import (
+	"context"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+)
+
+func (l Launchpad) ConstructionMetadata(ctx context.Context, r *types.ConstructionMetadataRequest) (*types.ConstructionMetadataResponse, *types.Error) {
+	if r.NetworkIdentifier == nil {
+		return nil, ErrInvalidRequest
+	}
+
+	if r.Options == nil {
+		return nil, ErrInvalidRequest
+	}
+
+	addr := r.Options[OptionAddress]
+	addrString := addr.(string)
```

Comment:
panics if `options` passed as an empty object.

Reply:
I think this will still panic. we need to check with an `ok`
`addr, ok := r.Options[OptionAddress]`

Reply:
yeah I think it panics.

Reply:
Ah yes, it'll be better to check if address is nil. Thanks

Reply:
you can do that

```
addr, ok := r.Options[OptionAddress]
if !ok {
return fmt,Err blahblha
}